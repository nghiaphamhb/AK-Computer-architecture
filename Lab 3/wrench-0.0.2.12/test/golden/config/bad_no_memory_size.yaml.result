"Aeson exception:
Error in $: parsing Wrench.Config.Config(Config) failed, key "memory_size" not found"
