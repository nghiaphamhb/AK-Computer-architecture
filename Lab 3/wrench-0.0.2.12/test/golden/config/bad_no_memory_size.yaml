input_streams:
  99: [1, 2, 3, "hello"]
limit: 0x90
